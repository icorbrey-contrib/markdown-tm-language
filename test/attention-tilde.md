Left letter:

ax~xb

ax~ b

ax~.b

Left whitespace:

a ~xb

a ~ b

a ~.b

Left punc:

a.~xb

a.~ b

a.~.b
