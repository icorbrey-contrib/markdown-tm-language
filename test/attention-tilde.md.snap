>Left letter:
#^^^^^^^^^^^^^ text.md meta.paragraph.md
>
>ax~xb
#^^ text.md meta.paragraph.md
#  ^ text.md meta.paragraph.md string.other.strikethrough.md
#   ^^^ text.md meta.paragraph.md
>
>ax~ b
#^^ text.md meta.paragraph.md
#  ^ text.md meta.paragraph.md string.other.strikethrough.md
#   ^^^ text.md meta.paragraph.md
>
>ax~.b
#^^ text.md meta.paragraph.md
#  ^ text.md meta.paragraph.md string.other.strikethrough.md
#   ^^^ text.md meta.paragraph.md
>
>Left whitespace:
#^^^^^^^^^^^^^^^^^ text.md meta.paragraph.md
>
>a ~xb
#^^ text.md meta.paragraph.md
#  ^ text.md meta.paragraph.md string.other.strikethrough.md
#   ^^^ text.md meta.paragraph.md
>
>a ~ b
#^^^^^^ text.md meta.paragraph.md
>
>a ~.b
#^^ text.md meta.paragraph.md
#  ^ text.md meta.paragraph.md string.other.strikethrough.md
#   ^^^ text.md meta.paragraph.md
>
>Left punc:
#^^^^^^^^^^^ text.md meta.paragraph.md
>
>a.~xb
#^^ text.md meta.paragraph.md
#  ^ text.md meta.paragraph.md string.other.strikethrough.md
#   ^^^ text.md meta.paragraph.md
>
>a.~ b
#^^ text.md meta.paragraph.md
#  ^ text.md meta.paragraph.md string.other.strikethrough.md
#   ^^^ text.md meta.paragraph.md
>
>a.~.b
#^^ text.md meta.paragraph.md
#  ^ text.md meta.paragraph.md string.other.strikethrough.md
#   ^^^ text.md meta.paragraph.md
>